diff -Naur synchronization-tools.orig//synchronization-tools.lisp synchronization-tools/synchronization-tools.lisp
--- synchronization-tools.orig//synchronization-tools.lisp	2011-02-10 09:32:58.000000000 +0100
+++ synchronization-tools/synchronization-tools.lisp	2011-07-29 15:58:25.000000000 +0200
@@ -10,6 +10,7 @@
                         #:with-local-interrupts
                         #:allow-with-interrupts)
   (:import-from :sb-ext #:atomic-incf #:atomic-decf)
+  (:shadow #:barrier)
   (:export
    ;; Buckets
    #:with-buckets
