--- src/assimp-grovel.lisp	2015-02-16 21:26:09.564876613 +0100
+++ src/assimp-grovel-12.04.lisp	2015-02-16 21:24:49.040878119 +0100
@@ -28,11 +28,11 @@
 ;;; POSSIBILITY OF SUCH DAMAGE.
 ;;;
 
-(include "assimp/cimport.h")
-(include "assimp/mesh.h")
-(include "assimp/scene.h")
-(include "assimp/material.h")
-(include "assimp/postprocess.h")
+(include "assimp/assimp.h")
+(include "assimp/aiMesh.h")
+(include "assimp/aiScene.h")
+(include "assimp/aiMaterial.h")
+(include "assimp/aiPostProcess.h")
 
 (in-package :physics-utils)
 
